_targets/build.project.armv8r52-mps3an536

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+#
+# Shell script for building armv8r52-mps3an536 project
+#
+# Copyright 2018-2020, 2024 Phoenix Systems
+# Author: Kaja Swat, Aleksander Kaminski, Pawel Pisarczyk, Lukasz Kosinski, Lukasz Leczkowski
+#
+[ "${BASH_SOURCE[0]}" -ef "$0" ] && echo "You should source this script, not execute it!" && exit 1
+
+. "_targets/build.common"
+
+CROSS=arm-phoenix-
+
+export BUSYBOX_CONFIG=$(realpath "busybox_config")
+export DROPBEAR_CUSTOM_CFLAGS="-DLTC_NO_BSWAP"
+export PSH_DEFUSRPWDHASH="0B1ANiYi45IhxkfmUW155/GBd4IRE="
+
+#
+# Ports configuration
+#
+export PORTS_BUSYBOX=n
+export PORTS_PCRE=n
+export PORTS_OPENSSL=n
+export PORTS_LIGHTTPD=n
+export PORTS_DROPBEAR=n
+export PORTS_LUA=n
+export PORTS_LZO=n
+export PORTS_OPENVPN=n
+export PORTS_JANSSON=n
+export PORTS_CURL=n
+
+
+#
+# Platform dependent parameters
+#
+SIZE_PAGE=$((0x200))
```

Comment:
Perhaps we can go with 0x1000 page size for this arch - small pages increase memory overhead when big amount of memory is available